<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataViewOptionsDock</class>
 <widget class="QDockWidget" name="DataViewOptionsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="showSondeDataButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="componentBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Time Filter</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Dataset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="startDateBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="showIntervalMissionButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Interval</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="showSingleMissionButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Single</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="startMissionSlider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="startMissionLabel">
           <property name="text">
            <string>12/01/2008</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>End</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="endMissionSlider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="endMissionLabel">
           <property name="text">
            <string>12/18/2008</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Selection</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Using:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="selTypeBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Select:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="selExpressionBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="selApplyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>203</width>
        <height>299</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
